src/gui/qjacktrip.ui

```diff
@@ -440,90 +339,275 @@ To connect to a hub server you need to run as a hub client.</string>
+             <property name="enabled">
+              <bool>false</bool>
+             </property>
              <property name="toolTip">
-              <string>Select how you want audio to be routed by the hub server.</string>
+              <string>Enter your username for the server.</string>
              </property>
-             <property name="currentIndex">
-              <number>0</number>
+            </widget>
+           </item>
+           <item row="4" column="0">
+            <widget class="QLabel" name="passwordLabel">
+             <property name="enabled">
+              <bool>false</bool>
+             </property>
+             <property name="text">
+              <string>&amp;Password</string>
+             </property>
+             <property name="buddy">
+              <cstring>passwordEdit</cstring>
              </property>
-             <item>
-              <property name="text">
-               <string>Server to clients</string>
-              </property>
-             </item>
-             <item>
-              <property name="text">
-               <string>Client loopback</string>
-              </property>
-             </item>
-             <item>
-              <property name="text">
-               <string>Client fan out/in but no loopback</string>
-              </property>
-             </item>
-             <item>
-              <property name="text">
-               <string>Full Mix</string>
-              </property>
-             </item>
-             <item>
-              <property name="text">
-               <string>No auto patching</string>
-              </property>
-             </item>
             </widget>
            </item>
-           <item row="0" column="0">
-            <widget class="QLabel" name="autoPatchLabel">
-             <property name="sizePolicy">
-              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
-               <horstretch>0</horstretch>
-               <verstretch>0</verstretch>
-              </sizepolicy>
+           <item row="3" column="0">
+            <widget class="QLabel" name="usernameLabel">
+             <property name="enabled">
+              <bool>false</bool>
              </property>
              <property name="text">
-              <string>Hub auto&amp;patch mode</string>
+              <string>&amp;Username</string>
              </property>
              <property name="buddy">
-              <cstring>autoPatchComboBox</cstring>
+              <cstring>usernameEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
-            <widget class="QCheckBox" name="patchServerCheckBox">
+            <widget class="QLabel" name="authNotVSLabel">
+             <property name="text">
+              <string>(This is for JackTrip's inbuilt authentication system. To easily connect to a Virtual Studio server, use the Virtual Studio interface.)</string>
```

Comment:
Suggestion as above 
`This is for JackTrip's inbuilt authentication system.` -> `This is for JackTrip's inbuilt authentication system, for the Virtual Studio.`
And:
`use the Virtual Studio interface` -> `use the Virtual Studio Mode` (that's the label of the button)

Reply:
How about this then? (Forgot the button was directly below because it's not in my builds.)
"(This is for JackTrip's inbuilt authentication system. To connect to a Virtual Studio server, press the Virtual Studio Mode button below.)"

Reply:
For the VS enabled build, it's probably less of an issue repeating Virtual Studio, so it could actually be:
(This is for JackTrip's inbuilt authentication system, not for the Virtual Studio. To connect to a Virtual Studio server, press the Virtual Studio Mode button below.)

Reply:
Thanks! I think the updated version is great.